src/main/java/core/basesyntax/service/FruitShopController.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import core.basesyntax.util.Injector;
+
+public class FruitShopController {
+    private static final String INPUT_FILE_PATH = "src/main/resources/reportToRead.csv";
+    private static final String OUTPUT_FILE_PATH = "src/main/resources/finalReport.csv";
```

Comment:
Avoid hardcode in your solution. It's better to pass file paths as parameters to the `run` method or to the constructor of `FruitShopController`, instead of using hardcoded paths. This will make your application more flexible and easier to test.